code-build.js

```diff
@@ -21,20 +21,23 @@ function runBuild() {
   const sdk = buildSdk();
 
   // Get input options for startBuild
-  const params = inputs2Parameters(githubInputs());
+  const [hideLog, p] = githubInputs();
 
-  return build(sdk, params);
+  const params = inputs2Parameters(p);
+
+  return build(sdk, params, hideLog);
 }
 
-async function build(sdk, params) {
+async function build(sdk, params, hideLog) {
   // Start the build
   const start = await sdk.codeBuild.startBuild(params).promise();
 
   // Wait for the build to "complete"
-  return waitForBuildEndTime(sdk, start.build);
+  return waitForBuildEndTime(hideLog, sdk, start.build);
 }
 
 async function waitForBuildEndTime(
+  hideLog,
```

Comment:
ditto - let's pass the hideLog inside the `config` object

https://github.com/aws-actions/aws-codebuild-run-build/blob/main/code-build.js#L41
https://github.com/aws-actions/aws-codebuild-run-build/blob/main/code-build.js#L47
https://github.com/aws-actions/aws-codebuild-run-build/blob/main/code-build.js#L106